modules/ibi-extra-manifests-standalone.adoc

```diff
@@ -21,6 +21,11 @@ You can create a resource in the `extra-manifests` folder of your working direct
 
 The following example adds an single-root I/O virtualization (SR-IOV) network to the deployment.
 
+[NOTE]
+====
+If you add more than one extra manifest, and the manifests must be applied in a specific order, you must prefix the filenames of the manifests with numbers representing the order in which they should be applied. For example, `00-namespace.yaml`, `01-sriov-extra-manifest.yaml`, and so on.
```

Comment:
```suggestion
If you add more than one manifest, to apply the manifests in a specific order, you must prefix the filenames of the manifests with numbers in the order you want to apply them. For example, `00-namespace.yaml`, `01-sriov-extra-manifest.yaml`, and so on.
```

- Suggested a change to avoid should. See: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s
- Suggested to _replace/passive-voice/active-voice_